src/hooks/useLoadAsync.ts

```diff
@@ -35,7 +35,8 @@ export const useLoadAsync = <T, P>(request: (params: P) => Promise<T>, params: P
 
   const load = useCallback(() => {
     loadAsync<T, P>(request, params, setData, setError, setLoading).catch(e => setError(e))
-    // TODO LUN-330 fix object comparison
+    /* For small objects JSON.stringify is good enough. But does not work with ESLint rule */
```

Comment:
since we don't send super large param objects `JSON.stringify`is fine. There are also other community options like this: https://github.com/kentcdodds/use-deep-compare-effect
But its not often updated